src/se/vidstige/jadb/JadbConnection.java

```diff
@@ -25,6 +25,20 @@ public Transport createTransport() throws IOException {
         return new Transport(new Socket(host, port));
     }
 
+    public void connect(String ip, int port) throws IOException, JadbException {
+        Transport main = this.createTransport();
+        main.send("host:connect:" + ip + ":" + port);
+        main.verifyResponse();
+        main.close();
+    }
```

Comment:
What does those method do? Forwards ip connection? Perhaps there can be better names?